### DISCUSSION TITLE
Ai to Png background transparency

### DISCUSSION THREAD

**Discussion by paulius-petkus** [dlemstra/Magick.NET#1729] [Help]
### Magick.NET version

14.0.0

### Environment (Operating system, version and so on)

Windows, .Net 6

### Description

Library fails to preserve transparent background when saving AI image to PNG. It becomes white.

We try to set background transparent:

```
image.Settings.BackgroundColor = MagickColors.Transparent;
using var fs = File.Create(Path.Combine(OutDirectory, $"{fileName}.{MagickFormat.ToString().ToLower()}"));
image.Write(fs, MagickFormat);
```

attaching zipped ai image:

[mhf09hehc8_iuj4n2.zip](https://github.com/user-attachments/files/17241595/mhf09hehc8_iuj4n2.zip)

### Steps to Reproduce

Save image as png.

**Comment by dlemstra** [comment]
This is happening due to a limitation of Ghostscript library. They don't support CMYKA and we will read the file as CMYK. You can work around this by reading the image in the `sRGB` colorspace:

```C#
var settings = new MagickReadSettings
{
    ColorSpace = ColorSpace.sRGB,
};

using var image = new MagickImage("mhf09hehc8_iuj4n2.ai", settings);
image.Write("mhf09hehc8_iuj4n2.png");
```

**Comment by paulius-petkus** [comment]
Thank you very much. I have just tested - works fine.
